pymc3/plots/__init__.py

```diff
@@ -14,17 +14,108 @@
 
 """PyMC3 Plotting.
 
-Plots are delegated to the `ArviZ <https://arviz-devs.github.io/arviz/>`_ library, a general purpose library for
-exploratory analysis of Bayesian models. For more details, see https://arviz-devs.github.io/arviz/.
-
-Only `plot_posterior_predictive_glm` is kept in the PyMC code base for now, but it will move to ArviZ once the latter adds features for regression plots.
+Plots are delegated to the ArviZ library, a general purpose library for
+"exploratory analysis of Bayesian models." See https://arviz-devs.github.io/arviz/
+for details on plots.
 """
 import functools
 import sys
 import warnings
 
 import arviz as az
 
+
+def map_args(func):
+    swaps = [("varnames", "var_names")]
```

Comment:
These should have been around for long enough - I replaced the soft with a hard warning.